### DISCUSSION TITLE
Harding Script

### DISCUSSION THREAD

**Discussion by eric-holtzclaw** [0x6d69636b/windows_hardening#55] [Ideas]
Do you have a or know how to lock down the ID numbers with a script? Registry keys and such?

**Comment by 0x6d69636b** [comment]
I don't understand what you mean, could you please elaborate with an example?

**Reply by eric-holtzclaw** [reply]
remediation script to harden the Windows OS?

**Reply by 0x6d69636b** [reply]
HardeningKitty can be used for this with the HailMary option: https://github.com/0x6d69636b/windows_hardening#hailmary
You can select one or more finding lists and then apply their recommendations to your system. 

I also recommend the backup function, so that you can restore the original settings if something goes wrong: https://github.com/0x6d69636b/windows_hardening#backup
